[MTM-46148]-DOC-BD-UI-Add-requirements-for-account-settings (#1035)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/changing-settings.md

- Provide [SMS provider credentials](#sms-provider).
- Manage the [connectivity settings](#connectivity).

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

> To see the **Authentication** menu entry, you must have "Tenant management" ADMIN permission.

Reviewer: According to @BeateRixen's comments on similar earlier pull requests, we should refer to "permissions" and "permission types", i.e.:
```suggestion
To see the **Authentication** menu entry, you must have ADMIN permission for the "Tenant management" permission type.
```

Author: I did a commit from Beate's suggestion.

> To see the **Authentication** menu entry, you must have "Tenant management" ADMIN permission.

Reviewer 2: ```suggestion
To see the **Authentication** menu item, you must have ADMIN permission for the "Tenant management" permission type.
```

> To see the **Authentication** menu item, you must have ADMIN permission for the "Tenant management" permission type.

Reviewer 3: ```suggestion
To see the **Authentication** menu entry, you must have ADMIN permission for the "Tenant management" permission type or be the first admin user created in the tenant.
```

Author: I followed Beate's suggestion
![MTM-46148_Beates_suggestion](https://github.com/SoftwareAG/c8y-docs/assets/133014746/8ff8f0d5-a1cc-44de-9c91-7cb9e76f798f)

Reviewer 3: updated suggestion to contain both, please consider it

Reviewer 2: > updated suggestion to contain both, please consider it

I agree, we should commit the suggestion from @Airdroplet.

> {{< /c8y-admon-req >}}

Author: I need more explanations here. 
There is `admin` global role which is needed to see `Authentication` menu item.  Which default roles info should be added here? 
![admins_global_role](https://github.com/SoftwareAG/c8y-docs/assets/133014746/7986ddd7-067f-4f5a-97a9-7990bf6a6429)

> {{< /c8y-admon-req >}}

Author: Is `Tenant Management` permission this default-thing which we are talking about?
![tenant_management_permission](https://github.com/SoftwareAG/c8y-docs/assets/133014746/d26a58be-0697-4f89-985c-338b4422af77)

> {{< /c8y-admon-req >}}

Reviewer: @Airdroplet, please confirm my understanding: we want to list the specialized (i.e. excluding admins, devices, business, readers) default global roles which contain the required permission(s) (in this case: ADMIN permission for the "Tenant management" permission type). I see you typically listed global role's name and description, and added a few words of introduction to the list, e.g.:
```
The above permissions can be used to create roles for robust user management. Every new tenant have specified typical roles by default:
- Tenant Manager - Can manage tenant wide configurations like applications, tenant options and retention rules.
```

> {{< /c8y-admon-req >}}

Author: ![Requirements_look](https://github.com/SoftwareAG/c8y-docs/assets/133014746/a42016d4-3881-4ff8-a79f-cc6dac32c893)

<a name="authentication"></a>
### Changing authentication settings

Click **Authentication** in the **Settings** menu if you want to view or change the Login or TFA settings.

![Authentication settings](/images/users-guide/Administration/admin-settings-authentication.png)

{{< c8y-admon-req >}}

> To see the **Authentication** menu entry, you must have "Tenant management" ADMIN permission (`ROLE_TENANT_ADMIN` or `ROLE_TENANT_MANAGEMENT_ADMIN`).

Reviewer: @Airdroplet, doesn't `ROLE_TENANT_ADMIN` mean that a user is the first created admin user for the tenant? This is separate from "ADMIN permission for the "Tenant management" permission type", so maybe we should add this to the info above, like:
```
To see the **Authentication** menu entry, you must have ADMIN permission for the "Tenant management" permission type or be the first admin user created in the tenant.
```
?

Author: ```be the first admin user created in the tenant```
Is that mean that users with admin role are not treated equally?

Author: I found old message in the chat from February 2022
```
My assumption is that the role TENANT_ADMIN is assigned only to the admin user created together with tenant and latter on is not assignable via UI
```
I don't know who can confirm it.

Reviewer 3: I think it is, it would not hurt to mention that in the way @pawel-rynarzewski-sag  suggested.

- Provide [SMS provider credentials](#sms-provider).
- Manage the [connectivity settings](#connectivity).

{{< c8y-admon-req >}}
ROLES & PERMISSIONS:

To see the **Authentication** menu item, you must have ADMIN permission for the "Tenant management" permission type or be the first admin user created in the tenant.

> The above permissions can be used to create roles for robust user management. Every new tenant have specified typical roles by default:

Reviewer 4: ```suggestion
The above permissions can be used to create roles for robust user management. Every new tenant has specific roles by default:
```

> - Tenant Manager - Can manage tenant wide configurations like applications, tenant options and retention rules.

Reviewer 4: ```suggestion
- Tenant manager - manages tenant-wide configurations like applications, tenant options and retention rules.
```
also, is this the only list item? the text above suggests that there's more than one role

Author: What do you mean by `list item`?

Reviewer: @rssag, yes, in this case it's the only item.

@pmich-sag, you could use the same sentence we came up with in the other pull request and use it here (adjusting singular/plural forms as needed):
```
For easier user access management, the above permission(s) are/is included in the global role(s) created by default in every new tenant:
```

Reviewer: @rssag, should we change letter case for "Tenant Manager" though? In the UI role names are displayed so::
![2023-05-19_15-37](https://github.com/SoftwareAG/c8y-docs/assets/92171763/3450a420-68bd-44ff-a0c6-34cf094d2c73)

Reviewer 4: updated suggestion

Author: Changed and pushed.

ROLES & PERMISSIONS:

To see the **Authentication** menu item, you must have ADMIN permission for the "Tenant management" permission type or be the first admin user created in the tenant.

The above permissions can be used to create roles for robust user management. Every new tenant have specified typical roles by default:

> - Tenant Manager - Can manage tenant wide configurations like applications, tenant options and retention rules.

Reviewer: I'd suggest to use the same phrasing we agreed on in https://github.com/SoftwareAG/c8y-docs/pull/1042/files#diff-910c4a9847d7827948109dd4046bac28ecc644eb1b73dae836dfd38dd0ec0f4aR21-R22.

Author: I did a change like Richard suggested in comment below.
